docs/config_options.md

```diff
@@ -159,12 +159,12 @@ If you define these options you will enable the associated feature, which may in
   * See [Ignore Mod Tap Interrupt](tap_hold.md#ignore-mod-tap-interrupt) for details
 * `#define IGNORE_MOD_TAP_INTERRUPT_PER_KEY`
   * enables handling for per key `IGNORE_MOD_TAP_INTERRUPT` settings
-* `#define TAPPING_FORCE_HOLD`
-  * makes it possible to use a dual role key as modifier shortly after having been tapped
-  * See [Tapping Force Hold](tap_hold.md#tapping-force-hold)
-  * Breaks any Tap Toggle functionality (`TT` or the One Shot Tap Toggle)
-* `#define TAPPING_FORCE_HOLD_PER_KEY`
-  * enables handling for per key `TAPPING_FORCE_HOLD` settings
+* `#define QUICK_TAP_TERM 100`
+  * double-tap and hold timing to use a dual role key to repeat keycode
+  * See [Quick Tap Term](tap_hold.md#quick-tap-term)
+  * Impacts any Tap Toggle functionality (`TT` or the One Shot Tap Toggle)
```

Comment:
"Breaks any Tap Toggle functionality' was more telling than "impacts". Using "impacts" just informs me that quick tap term does *things* to tap toggle functionality but no clue what. Does it change how many times I need to tap the key for it to toggle? Does it change how fast I must tap it? Does it break this functionality?